requirements-test.txt

```diff
@@ -0,0 +1,4 @@
+coverage
+pandas
```

Comment:
We don't rely on `pandas` for our Python test suite.

Reply:
good catch, fixed